pkg/values/packagemaps.go

```diff
@@ -117,13 +117,13 @@ var ImmucorePackages = PackageMap{
 var KernelPackages = PackageMap{
 	Ubuntu: {
 		ArchCommon: {
-			">=20.04, != 24.10, != 25.04, != 25.10": {
+			"20.04 || 22.04 || 24.04 || 26.04 || 28.04": {
 				// Note: this logic only works for LTS release (x.04), any odd year or x.10 will not work
 				// This is a template, so we can replace the version with the actual version of the system
 				"linux-image-generic-hwe-{{.version}}",
 			},
 			// 24.10 uses the 24.04 hwe kernel as it is the same hwe track https://ubuntu.com/kernel/lifecycle
-			"24.10 || 25.04 || 25.10": {"linux-image-generic-hwe-24.04"},
+			">=24.10": {"linux-image-generic-hwe-24.04"},
```

Comment:
At some point in time, this will match 26.04 as well 😭

Maybe it's time to move to a regex just for Ubuntu